updater/internal/install/install.go

```diff
@@ -60,31 +65,35 @@ func (i Installer) Install(rb rollback.ActionAppender) error {
 		return fmt.Errorf("failed to stop service: %w", err)
 	}
 	rb.AppendAction(action.NewServiceStopAction(i.svc))
+	i.logger.Debug("Service stopped")
 
 	// install files that go to installDirPath to their correct location,
 	// excluding any config files (logging.yaml, config.yaml, manager.yaml)
-	if err := copyFiles(i.latestDir, i.installDir, i.tmpDir, rb); err != nil {
+	if err := i.copyFiles(i.latestDir, i.installDir, i.tmpDir, rb); err != nil {
 		return fmt.Errorf("failed to install new files: %w", err)
 	}
+	i.logger.Debug("Install artifacts copied")
 
 	// Update old service config to new service config
 	if err := i.svc.Update(); err != nil {
 		return fmt.Errorf("failed to update service: %w", err)
 	}
-	rb.AppendAction(action.NewServiceUpdateAction(i.tmpDir))
+	rb.AppendAction(action.NewServiceUpdateAction(i.logger, i.tmpDir))
+	i.logger.Debug("Updated service configuration")
 
 	// Start service
 	if err := i.svc.Start(); err != nil {
 		return fmt.Errorf("failed to start service: %w", err)
 	}
 	rb.AppendAction(action.NewServiceStartAction(i.svc))
+	i.logger.Debug("Service started")
 
 	return nil
 }
 
 // copyFiles moves the file tree rooted at latestDirPath to installDirPath,
 // skipping configuration files. Appends CopyFileAction-s to the Rollbacker as it copies file.
-func copyFiles(inputPath, outputPath, tmpDir string, rb rollback.ActionAppender) error {
+func (i Installer) copyFiles(inputPath, outputPath, tmpDir string, rb rollback.ActionAppender) error {
```

Comment:
This is fine making it a receiver on the object but seems like if this is just for access to the logger could we just pass the logger in?